checks/raw/permissions.go

```diff
@@ -491,6 +491,16 @@ func isReleasingWorkflow(workflow *actionlint.Workflow, fp string, pdata *permis
 			},
 			LogText: "candidate SLSA publishing workflow using slsa-github-generator",
 		},
+		{
+			// Running mvn release:prepare requires committing changes.
+			// https://maven.apache.org/maven-release/maven-release-plugin/examples/prepare-release.html
+			Steps: []*fileparser.JobMatcherStep{
+				{
+					Run: ".*mvn.*release:prepare.*",
```

Comment:
should this be `mvnw` instead of `mvn`?

Reply:
Not everyone uses mvnw; some people just depend on an installed mvn instead. The regex will allow both to work

Reply:
Got it, thanks.